packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -74,12 +75,15 @@ library LibVerifying {
         TaikoData.Config memory config,
         AddressResolver resolver,
         uint256 maxBlocks
-    ) internal {
+    )
+        internal
+    {
         uint256 i = state.lastVerifiedBlockId;
         TaikoData.Block storage blk = state.blocks[i % config.ringBufferSize];
 
         uint256 fcId = blk.verifiedForkChoiceId;
-        assert(fcId > 0);
+        // assert(fcId > 0);
+        require(fcId > 0, "ABC");
```

Comment:
is this "ABC" error message what we want?

Reply:
Removed